From 572e59f69b519b2525637e27e229a22d573df10d Mon Sep 17 00:00:00 2001
From: Marcelo Tosatti <mtosatti@redhat.com>
Date: Mon, 1 Mar 2010 20:52:07 -0300
Subject: [PATCH 04/20] Increase VNC_MAX_WIDTH

RH-Author: Marcelo Tosatti <mtosatti@redhat.com>
Message-id: <664ed8c134b6fc1c44599588a55c1d7bb37fe50a.1267476474.git.mtosatti@redhat.com>
Patchwork-id: 7354
O-Subject: [PATCH 05/21] Increase VNC_MAX_WIDTH
Bugzilla: 569613
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Glauber Costa <glommer@redhat.com>
RH-Acked-by: Gleb Natapov <gleb@redhat.com>

From: Brian Jackson <iggy@theiggy.com>

Increase VNC_MAX_WIDTH to match "commonly available" consumer level monitors
available these days.

This also closes KVM bug 2907597

Signed-off-by: Brian Jackson <iggy@theiggy.com>
Signed-off-by: Anthony Liguori <aliguori@us.ibm.com>
Signed-off-by: Avi Kivity <avi@redhat.com>
(cherry picked from commit ded993f117fe7020af319ad8903e9f07bf35dc71)
---
 vnc.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 vnc.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/vnc.h b/vnc.h
index 36e6bfc..71280cb 100644
--- a/vnc.h
+++ b/vnc.h
@@ -68,7 +68,7 @@ typedef void VncSendHextileTile(VncState *vs,
                                 void *last_fg,
                                 int *has_bg, int *has_fg);
 
-#define VNC_MAX_WIDTH 2048
+#define VNC_MAX_WIDTH 2560
 #define VNC_MAX_HEIGHT 2048
 #define VNC_DIRTY_WORDS (VNC_MAX_WIDTH / (16 * 32))
 
-- 
1.7.0.3

